symbols.go

```diff
@@ -14,28 +12,16 @@ const (
 )
 
 func (s LogSymbol) String() string {
-	if runtime.GOOS == "windows" {
```

Comment:
This was supposed to be `runtime.GOOS != "windows"` because the symbols don't work on windows.

Reply:
Ah, got it :)